modules/remoting.nu

```diff
@@ -1,136 +1,72 @@
-#?name: remoting
+#?name: make
```

Comment:
Please get the newest remoting.nu and make.nu before merging

Reply:
Done